eahub/templates/eahub/newsletter.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<link href="/static/styles/mailchimp.css" rel="stylesheet" type="text/css">
+
+<div class="panel panel-default">
+<div class="panel-body">
+<div id="mc_embed_signup">
+<form action="https://effectivealtruismhub.us11.list-manage.com/subscribe/post?u=48139d2c8cadf4a1bce00bb6f&amp;id=7051ba2a99" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+<div id="mc_embed_signup_scroll">
+	<h2>Subscribe to our Newsletter</h2>
+  <div class="col-md-4 col-md-offset-4">
+  <div class="mc-field-group field">
+  	<label for="mce-EMAIL">Email Address  <span>*</span></label>
+  	<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
```

Comment:
Can we prefill the fields if the user's logged in? I wouldn't ask if it was going to be hard, but I think it should be easy.